### PR TITLE
Fix integration test workflow for windows

### PR DESCRIPTION
**Pull request by vsiddharth**
### Summary

Fix integration test workflow for windows

### Implementation details

This commit updates the script used to invoke the integration test.
With the upgrade to golang v1.17, the tests expect to be invoked from
where the go module file is located.

### Testing

Verified changes manually on Windows environment. 

New tests cover the changes:  No

### Licensing

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by sharanyad — APPROVED** [review]

File: scripts/run-integ-tests.ps1 [hunk 1]
```diff
@@ -46,8 +46,10 @@ Invoke-Expression "${PSScriptRoot}\..\misc\netkitten\build.ps1"
 # Run the tests
 $cwd = (pwd).Path
 try {
-  cd "${PSScriptRoot}"
   $env:ECS_LOGLEVEL = 'debug'; go test -race -tags integration -timeout=40m -v ../agent/engine ../agent/stats ../agent/app
+  if (${LastExitCode} -ne 0) {
```
Comment (rawahars) [inline scripts/run-integ-tests.ps1:50]: Can we create a loop counter to repeat the tests?

Example-
```
$count =2
for(;$count -ne 0;$count--)
{
    if (${LastExitCode} -ne 0) {
        $env:ECS_LOGLEVEL = 'debug'; go test -race -tags integration -timeout=40m -v ../agent/engine ../agent/stats ../agent/app
    }
}
```

**Comment by rawahars** [comment]
Please rebase over the dev. The integration test - TestMultipleDynamicPortForward and TestDynamicPortForward should pass once this commit is included in the PR.

File: scripts/run-integ-tests.ps1 (hunk 1 above)
Comment (vsiddharth) [reply scripts/run-integ-tests.ps1:50]: > Can we create a loop counter to repeat the tests?
> 
> Example-
> 
> ```
> $count =2
> for(;$count -ne 0;$count--)
> {
>     if (${LastExitCode} -ne 0) {
>         $env:ECS_LOGLEVEL = 'debug'; go test -race -tags integration -timeout=40m -v ../agent/engine ../agent/stats ../agent/app
>     }
> }
> ```

Why do we need `$count =2` ? Shouldn't running this optionally one another time be sufficient?

**Comment by vsiddharth** [comment]
> Please rebase over the dev. The integration test - TestMultipleDynamicPortForward and TestDynamicPortForward should pass once this commit is included in the PR.

Done!

File: scripts/run-integ-tests.ps1 (hunk 1 above)
Comment (rawahars) [reply scripts/run-integ-tests.ps1:50]: I have updated the code snippet above to break on the first try if the tests succeed. If not, the tests are executed one more time.

**Review by fenxiong — APPROVED** [review]

**Review by sharanyad — APPROVED** [review]

**Pull request merged**